tests/bootstrap.php

```diff
@@ -17,7 +17,7 @@
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
 AnnotationRegistry::registerLoader(array($autoload, 'loadClass'));
-AnnotationRegistry::registerFile(__DIR__.'/../lib/Doctrine/ODM/PHPCR/Mapping/Annotations/DoctrineAnnotations.php');
+//AnnotationRegistry::registerFile(__DIR__.'/../lib/Doctrine/ODM/PHPCR/Mapping/Annotations/DoctrineAnnotations.php');
```

Comment:
Removed the line. I'm not sure about removing registerLoader. I know Symfony registers `class_exists` as an annotation loader, so it kind of  seems like you need to configure at least one loader to make things work. Let's do it in another PR...